Update DOS code formatting in numbered list

ned1313 opened the pull request:
Attempting to fix the way that the DOS code is formatted in the numbered list.  On the MS site it loses the newlines and the DOS tag for the code.

AndrewHoh commented:
@mimig1

mimig1 commented:
@AndrewHoh - Can you submit this to azure-content-pr instead so we can see it on staging first? With so many changes, I think that's a safer bet.

mimig1 commented:
@AndrewHoh please close this if it went through azure-content-pr instead. thx.

ned1313 committed:
Update DOS code formatting in numbered list

Attempting to fix the way that the DOS code is formatted in the numbered list.  On the MS site it loses the newlines and the DOS tag for the code.
